pvlib/ivtools/fit_sdm.py

```diff
@@ -0,0 +1,1087 @@
+"""
+The ``fit_sdm`` module contains functions to fit single diode models.
+
+Function names should follow the pattern "fit_sdm_" + name of model + "_" +
+ fitting method.
+
+"""
+
+import numpy as np
+from collections import OrderedDict
+
+import logging
+
+from pvlib.pvsystem import singlediode, v_from_i
+
+from utility import constants, rectify_iv_curve, numdiff
+from fit_sde import fit_sde_cocontent
```

Comment:
both of these need to be full namescape paths starting from `pvlib` or using leading dots.